cmd/apiserver_receive_adapter/main.go

```diff
@@ -33,6 +33,7 @@ const (
 func main() {
 	ctx := signals.NewContext()
 	ctx = adapter.WithInjectorEnabled(ctx)
+	ctx = adapter.WithHealthProbesDisabled(ctx)
```

Comment:
I see that the dynamically created ApiServer receive adapter defines the readiness probe here: https://github.com/knative/eventing/blob/bff7b031831fd7968d86e99ae89d79cf3f088eed/pkg/reconciler/apiserversource/resources/receive_adapter.go#L98-L108

why are we disabling the server here ? is the ApiServer receive adapter still using a different server?

Reply:
Commit message doesn't help explaining the reasoning but I think there is one and I'm curious on your reasoning: https://github.com/knative/eventing/pull/8308/commits/224b08db3643de356ba3f1fbcd8659f7ae625306

Reply:
Yeah,  the ApiServer receive adapter has its own server here: https://github.com/knative/eventing/blob/bff7b031831fd7968d86e99ae89d79cf3f088eed/pkg/adapter/apiserver/adapter.go#L129-L136

Should we replace this as well?